content/applications/services/helpdesk/overview/sla.rst

```diff
@@ -2,183 +2,209 @@
+   Employees --> Work Organization`.
+
+The deadline is then added to the ticket, as well as a tag indicating the name of the :abbr:`SLA
+(Service Level Agreement)` applied.
 
 .. image:: sla/sla-open-deadline.png
    :align: center
-   :alt: View of a ticket's form emphasizing an open SLA deadline on a ticket in Odoo Helpdesk
+   :alt: View of a ticket's form emphasizing an open SLA deadline on a ticket in Odoo Helpdesk.
+
+When a ticket satisfies an :abbr:`SLA (Service Level Agreement)` policy, the :abbr:`SLA (Service
+Level Agreement)` tag turns green, and the :guilabel:`Deadline` disappears from view on the ticket.
 
 .. important::
    If a ticket fits the criteria for more than one :abbr:`SLA (Service Level Agreement)`, the
-   earliest occurring deadline will be displayed on the ticket. Once that deadline has passed, the
+   earliest occurring deadline will be displayed on the ticket. After that deadline has passed, the
    next deadline will be displayed.
 
-Once a ticket satisfies an :abbr:`SLA (Service Level Agreement)` policy, the :abbr:`SLA (Service
-Level Agreement)` tag turns green, and the :guilabel:`Deadline` field disappears from view on the
-ticket.
-
 .. image:: sla/sla-deadline.png
    :align: center
-   :alt: View of a ticket's form emphasizing a satisfied SLA in Odoo Helpdesk
+   :alt: View of a ticket's form emphasizing a satisfied SLA in Odoo Helpdesk.
 
 If the :abbr:`SLA (Service Level Agreement)` deadline passes and the ticket has not moved to the
-:guilabel:`Reach Stage`, the :abbr:`SLA (Service Level Agreement)` tag will turn red. Once the
+:guilabel:`Reach Stage`, the :abbr:`SLA (Service Level Agreement)` tag will turn red. After the
 :abbr:`SLA (Service Level Agreement)` has failed, the red tag will stay on the ticket, even after
 the ticket is moved to the :guilabel:`Reach Stage`.
 
 .. image:: sla/sla-passing-failing.png
    :align: center
-   :alt: View of a ticket's form with a failing and passing SLA in Odoo Helpdesk
+   :alt: View of a ticket's form with a failing and passing SLA in Odoo Helpdesk.
 
 Analyzing SLA performance
 =========================
 
 The :guilabel:`SLA Status Analysis` report tracks how quickly an :abbr:`SLA (Service Level
-Agreement)` is fulfilled, as well as the success rate of individual policies. Navigate to the report
-and corresponding pivot table by going to :menuselection:`Helpdesk --> Reporting --> SLA Status
-Analysis`.
+Agreement)` is fulfilled, as well as the performance of individual team members. Navigate to the
+report and corresponding pivot table by going to :menuselection:`Helpdesk app --> Reporting --> SLA
+Status Analysis`.
 
 Using the Pivot view
 --------------------
 
-By default, the report is displayed in a :guilabel:`Pivot` view, and is filtered to show the number
-of SLAs failed and the failure rate over the last 30 days, grouped by team.
+By default, the report displays in a :guilabel:`Pivot` view. Any :abbr:`SLA (Service Level
```

Comment:
using GUI for the Pivot view in this case seems admissible because that's the name of the button when the cursor hovers over it, unless you think otherwise...